docs/concepts/api-extension/custom-resources.md

```diff
@@ -62,16 +101,100 @@ and setup a controller to handle events.
+| Defaulting | See above | No, but can achieve the same effect with an Initializer (requires programming) | Yes |
+| Multi-versioning | Allows serving the same object through two API versions. Can help ease API changes like renaming fields. Less important if you control your client versions. | No | Yes |
+| Custom Storage | If you need storage with a different performance mode (for example, time-series database instead of key-value store) or isolation for security (for example, encryption secrets or different | No | Yes |
+| Custom Business Logic | Perform arbitrary checks or actions when creating, reading, updating or deleting an object | No, but can get some of the same effects with Initializers or Finalizers (requires programming) | Yes |
+| Subresources | <ul><li>Add extra operations other than CRUD, such as "scale" or "exec"</li><li>Allows systems like like HorizontalPodAutoscaler and PodDisruptionBudget interact with your new resource</li><li>Allows splitting responsibility for setting spec vs status</li><li>Allows incrementing object Generation on custom resource data mutation (requires Spec/Status split)</li></ul> | No but planned | Yes, any Subresource |
+| strategic-merge-patch | The new endpoints support PATCH with `Content-Type: application/strategic-merge-patch+json`. Useful for updating objects that may be modified both locally, and by the server. For more information, see ["Update API Objects in Place Using kubectl patch"](/docs/tasks/run-application/update-api-object-kubectl-patch/) | No | Yes |
+| Protocol Buffers | The new resource supports clients that want to use Protocol Buffers | No | Yes |
+| OpenAPI Schema | Is there an OpenAPI (swagger) schema for the types that can be dynamically fetched from the server? Is the user protected from misspelling field names by ensuring only allowed fields are set? Are types enforced (in other words, don't put an `int` in a `string` field?) | No but planned | Yes |
+
+#### Common Features
+
+CRDs and AAs support many of the same features:
+
+| Feature | What it does |
+|-|-|
+| CRUD | The new endpoints support CRUD basic operations via HTTP |
+| Watch | The new endpoints support Kubernetes Watch operations via HTTP |
+| Discovery | Clients like kubectl and dashboard automatically offer list, display, and field edit operations on your resources |
+| json-patch | The new endpoints support PATCH with `Content-Type: application/json-patch+json` |
+| merge-patch | The new endpoints support PATCH with `Content-Type: application/merge-patch+json` |
+| HTTPS | The new endpoints uses HTTPS |
+| Built-in Authentication | Can the extension reuse the core apiserver's authentication? |
```

Comment:
These and the following lines need to be resolved with actual descriptions, and not questions.